registry-manager-db-api/src/main/java/de/adorsys/registry/manager/repository/AspspRepository.java

```diff
@@ -61,4 +61,11 @@
      * @param aspsps, a list of objects
      */
     void delete(List<AspspPO> aspsps);
+
+    /**
+     * Counts total number of ASPSPs entities persisting in the database
+     *
+     * @return total number of elements in the repository
+     */
+    Long count();
```

Comment:
I'd propose to go with `long` instead of `Long` (of course, if it is suitable for your case), because as far as I see, Spring returns `long` as well (means, that `null` is not possible there)

Reply:
done